frontend/src/Components/ContainerVhVw/ContainerVhVw.tsx

```diff
@@ -6,6 +6,9 @@ interface IContainerVhVw {
   children: ReactNode;
 }
 
+/**
+ * @deprecated Use CenterContainer instead
+ */
```

Comment:
if it's deprecated - why are we using it?

Reply:
It was a first version of this component. I guess we can keep it and mention it in our thesis.